src/panels/lovelace/custom-card-helpers.ts

```diff
@@ -3,3 +3,9 @@ export { createCardElement } from "./create-element/create-card-element";
 export { createBadgeElement } from "./create-element/create-badge-element";
 export { createHeaderFooterElement } from "./create-element/create-header-footer-element";
 export { createHuiElement } from "./create-element/create-hui-element";
+export { actionHandlerBind } from "./common/directives/action-handler-directive";
+export { handleAction } from "./common/handle-action";
+
+export const loadElements = {
```

Comment:
What is this future proofing for? Are we just going to keep putting things in here that some custom card might need ?

Reply:
I hope not, but if we are, are we just going to throw them in without any grouping?

Reply:
Excellent question 😄 I don't know.

Reply:
I agree that we are postponing loading a bunch of things, because loading a custom card shouldn't just load half of the Lovelace code just to sit unused.

Reply:
Shouldn't that also apply to the action handler?

Reply:
There's not much that doesn't use the action handler.

Reply:
The action handler is lazy loaded by the actionHandlerBind function.